src/HLSLTree.cpp

```diff
@@ -1409,139 +1409,5 @@ void HideUnusedArguments(HLSLFunction * function)
     }
 }
 
-
-class FindFunctionCallVisitor : public HLSLTreeVisitor
```

Comment:
P.S. In the original codebase FindFunctionCall performs reachability traversal from the entry point. This may become impossible if we ever add support for multiple entrypoints in one shader source, and this becomes unnecessary if you really don't need the code - you can easily call PruneTree and pass it the entrypoint. I must confess I haven't tested the MSL codegen changes in our compilation stack without pruning - I'll do that to see if there are any correctness issue with the non-pruned tree generation.